shared/src/api/extension/extensionHost.ts

```diff
@@ -195,10 +198,12 @@ function createExtensionAPI(
             rootChanges: roots.changes,
         },
 
-        configuration: {
-            get: () => configuration.get(),
-            subscribe: configuration.subscribe.bind(configuration),
-        },
+        configuration: Object.assign(
+            new Observable<void>(observer => configuration.changes.subscribe(observer)),
```

Comment:
There's an app for that ™️ 
```suggestion
            configuration.changes.asObservable(),
```